pkg/bigqueryexporter/bigquery.go

```diff
@@ -73,15 +76,18 @@ func (client *Client) ExportReleaseTags(ctx context.Context, dbClient *db.DB) er
 		rows = append(rows, row)
 	}
 
-	return dbClient.DB.Clauses(clause.OnConflict{UpdateAll: true}).Create(&rows).Error
+	return dbClient.DB.Clauses(clause.OnConflict{UpdateAll: true}).CreateInBatches(&rows, dbClient.BatchSize).Error
 }
 
 func (client *Client) ExportPullRequests(ctx context.Context, dbClient *db.DB) error {
 	rows := make([]models.PullRequest, 0)
-	query := client.Query(`
-		SELECT
-			ROW_NUMBER() OVER() id, *
-		FROM ` + "`ci_data.ReleasePullRequests`")
+	query := client.Query(`SELECT * FROM (SELECT ROW_NUMBER() OVER() id, * FROM ` + "`ci_data.ReleasePullRequests`) WHERE id > @lastID")
```

Comment:
So bigquery literally has no ID we can reuse on these rows in the table?

Reply:
[Nope](https://cloud.google.com/blog/products/data-analytics/bigquery-and-surrogate-keys-practical-approach). We could create and manage our own, but ROW_NUMBER seems to work ok.